### ISSUE TITLE
Gutter causes column to wrap in MS Edge

### ISSUE THREAD

**Issue by RaptorCZ** [twbs/bootstrap#19363]
Hi, I have my own compiled version of Bootstrap with modified settings. I need to set small gutter instead of 30px (in variables.less).

So when @grid-gutter-width = 4px, grid system doesn't work in some scenarios in MS Edge. Tried 6px or 0 and both works fine. But 4px is broken.

Example with gutter = 4px
https://jsfiddle.net/RaptorCZ/upx52moL/

Example with gutter = 6px
https://jsfiddle.net/RaptorCZ/1rdojhy0/

Simply set result window to be as wide as possible (to prevent breaks on lower viewport) and 2x click on 2nd panel to collapse and expand.

When gutter is 4px, 3rd label collapses under label 1. Why?
![bootstrap_gutter_4](https://cloud.githubusercontent.com/assets/419884/14043171/96af4fb6-f23e-11e5-8d87-88be3526fc81.jpg)

It is ok when gutter = 6px.
![bootstrap_gutter_6](https://cloud.githubusercontent.com/assets/419884/14043183/a75c8d1a-f23e-11e5-837d-f41ca7993c74.jpg)

**Comment by twbs-lmvtfy** [comment]
Hi @RaptorCZ!

You appear to have posted a live example (https://fiddle.jshell.net/RaptorCZ/upx52moL/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 6858, column 27: [E042](https://github.com/twbs/bootlint/wiki/E042): `.form-control` cannot be used on non-textual `<input>`s, such as those whose `type` is: `file`, `checkbox`, `radio`, `range`, `button`
* line 6870, column 27: [E042](https://github.com/twbs/bootlint/wiki/E042): `.form-control` cannot be used on non-textual `<input>`s, such as those whose `type` is: `file`, `checkbox`, `radio`, `range`, `button`
* line 6882, column 27: [E042](https://github.com/twbs/bootlint/wiki/E042): `.form-control` cannot be used on non-textual `<input>`s, such as those whose `type` is: `file`, `checkbox`, `radio`, `range`, `button`

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by twbs-lmvtfy** [comment]
Hi @RaptorCZ!

You appear to have posted a live example (https://fiddle.jshell.net/RaptorCZ/1rdojhy0/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 6857, column 27: [E042](https://github.com/twbs/bootlint/wiki/E042): `.form-control` cannot be used on non-textual `<input>`s, such as those whose `type` is: `file`, `checkbox`, `radio`, `range`, `button`
* line 6869, column 27: [E042](https://github.com/twbs/bootlint/wiki/E042): `.form-control` cannot be used on non-textual `<input>`s, such as those whose `type` is: `file`, `checkbox`, `radio`, `range`, `button`
* line 6881, column 27: [E042](https://github.com/twbs/bootlint/wiki/E042): `.form-control` cannot be used on non-textual `<input>`s, such as those whose `type` is: `file`, `checkbox`, `radio`, `range`, `button`

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by RaptorCZ** [comment]
Errors fixed (I hope), problem persists :-)

**Comment by cvrebert** [comment]
Reposting to trigger revalidation:
https://jsfiddle.net/RaptorCZ/upx52moL/
https://jsfiddle.net/RaptorCZ/1rdojhy0/

**Comment by cvrebert** [comment]
@RaptorCZ Have you tested in non-Edge browsers?

**Comment by RaptorCZ** [comment]
Yes, IE 11, Chrome and FF works fine. Seems that only Edge is problem.

Maybe the problem has something to do with using -ms-overflow-style: scrollbar. Default behavior in Edge is -ms-autohiding-scrollbar.

Column wraps only when scrollbar is visible.

**Comment by RaptorCZ** [comment]
Any news on this, or any temporary workaround?

**Comment by mdo** [comment]
[**Bootstrap 3 is no longer being officially developed or supported.**](https://github.com/twbs/bootstrap/issues/20631)

All work has moved onto our next major release, v4. As such, this issue or pull request is being closed as a "won't fix." For additional help and support, we recommend [utilizing our community resources](https://github.com/twbs/bootstrap/blob/master/README.md#community). Thanks for your understanding, and see you on the other side of v4!

<3,
@mdo and team
